pages/Easter-Seasonal-Category.md

```diff
@@ -0,0 +1,14 @@
+The Easter Seasonal Category is a category only available in the month around Easter (April).  
+
+## Table of contents
+This category only contains 3 items, under which the **Carrot Pie** and **Apple Pie**, which are simple types of foods that you can craft and then, of course, eat.  
+
+It also contains an **Easter Egg**, which you can open and it will leave a little surprise.  
+
+#### It can drop the following items:
```

Comment:
Since I am picky (thank Walshy), this is a little misleading. It stands out compared to the above so it seems like it would refer to all three or something different and starts with "it"
I would change it to something like
```suggestion
#### Easter Egg surprises list:
```
Or something